FrontEnd/Brewerweb/js/CheckFields.js

```diff
@@ -0,0 +1,127 @@
+/* Justin Evans
+*/
+// This script validates a form.
+
+// Function called when the form is submitted.
+// Function validates the form data.
+function validateForm(e) {
+    'use strict';
+
+    // Get the event object:
+	if (typeof e == 'undefined') e = window.event;
+
+    // Get form references:
+	var firstName = U.$('firstName');
+	var lastName = U.$('lastName');
+	var email = U.$('email');
+	var phone = U.$('phone');
+	var city = U.$('city');
+	var state = U.$('state');
+	var zip = U.$('zip');
+	var terms = U.$('terms');
+
+	// Flag variable:
+	var error = false;
+
+	// Validate the first name:
+	if (/^[A-Z \.\-']{2,20}$/i.test(firstName.value)) {

```

Comment:
Maybe condense `/^[A-Z \.\-']{2,20}$/i` into a variable since it is used again for last name and for city.